api/v1alpha1/envoypatchpolicy_types.go

```diff
@@ -19,7 +19,8 @@ const (
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:categories=envoy-gateway,shortName=epp
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[?(@.type=="Programmed")].reason`
+// +kubebuilder:printcolumn:name="Accepted",type=string,JSONPath=`.status.ancestors[0].conditions[?(@.type=="Accepted")].status`
```

Comment:
why we only check the first ancestors?

Reply:
it's fine to me use `ancestors[0]`, it's should be accepted for all or none.

@envoyproxy/gateway-maintainers PTAL